pages/currency-exchange/currency-exchange.module.css

```diff
@@ -0,0 +1,16 @@
+.exchange_rates {
+  margin: 5%;
+}
+
+@media only screen and (max-width: 1200px) {
+  /*Tablets [601px -> 1200px]*/
+}
```

Comment:
Why is this needed when this does not contain any css in it?
same for line no 14-16

Reply:
we would probably need, I am still testing the views, this is just the initial mr to show how this looks at this moment

Reply:
Sure, in that case can you please add a WIP tag to this PR?